### ISSUE TITLE
force AC3 transcode for all content

### ISSUE THREAD

**Issue by mitchelp** [croneter/PlexKodiConnect#335]
I got some trouble setting the audio output for Kodi/PlexKodiConnect correct. What I would like to achieve is the following:
Got my Mac Mini hooked up to my Samsung KS8000 TV. This one is configured to Bitstream and Dolby Digital (if available).
This all works good for my Chromecast ultra. That is playing every audio source fine. It seems to convert everything to AC3 (Dolby Digital) 

- DTS --> Dolby Digital
- True-HD --> Dolby Digital
- etc. etc.

So what I want Kodi (or PMS) to do is the following:
Convert every single audio track to Dolby Digital. Cause this is the only codec that my Sonos Playbar will understand and my TV will passthrough and play correctly.

if I pick a video source already equipped with Dolby Digital it's working fine.

Hoping to get some help around here for what i'm doing wrong.

Thanks already!

**Comment by croneter** [comment]
Sorry, this is not an "issue"/bug, but a "feature request". Please open a new one here and be as specific as possible: http://feathub.com/croneter/PlexKodiConnect

Workaround: use the Kodi context menu, then Plex options, then `Force transcode`

**Comment by croneter** [comment]
Actually, you might be able to simple configure Kodi to transcode audio. Check with http://kodi.wiki/view/AudioEngine/HOW-TO:_Configure_audio

**Comment by mitchelp** [comment]
@croneter I think I figured it out.. My god I feel stupid. Took me a few days..
Problem was the Channels (have to confirm it this afternoon).

My TV is connect over HDMI. But my soundbar is over SPDIF. And SPDIF only supports 2 PCM channels. What means If I configure kodi to send it over 5.1 channels. It will send 6PCM streams to my TV and because my TV can't send it over PCM only stereo will get to my soundbar.

What means. that:
I had to set kodi to 2.0 channels.
Configure the passthrough settings to AC3 and AC3 transcode. (what only is visible when you configure kodi to 2.0 channels). What will send Dolby Digital AC3 over 2PCM channels what is more than enough space for 5.1 channels what hopefully my soundbar can pickup from there.
